net-misc/spice-gtk/spice-gtk-0.40.ebuild

```
@@ -94,6 +97,7 @@ python_check_deps() {
 src_prepare() {
 	default
 
+	use elibc_musl && (eapply "${FILESDIR}"/${PN}-0.40-support_libucontext.patch || die)
```

1. No need for subshell (the parentheses)
2. No need for die with eapply, it "dies for us" as a builtin

So..
```suggestion
	use elibc_musl && apply "${FILESDIR}"/${PN}-0.40-support_libucontext.patch
```

But actually, you should be able to apply this unconditionally safely.

---

(See also: https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#conditional-patching. It's okay if we have to, but when it's avoidable, we shouldn't - because e.g. it might stop applying and nobody notices)

---

oh, avoiding conditional patches makes sense then. I was a little unsure what was best when I made the patch.